client/src/components/Header.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <header>
+  <header id="header">
```

Comment:
```suggestion
  <header>
```
If `id="header"` is not used anymore, we can remove it